src/client/components/icons/Icon.tsx

```diff
@@ -0,0 +1,24 @@
+import type { PropsWithChildren } from "preact/compat";
+
+export interface IconProps {
+  size?: string;
+  className?: string;
+}
+export default function Icon({
+  size = "w-5 h-5",
+  className = "",
+  children,
+}: PropsWithChildren<IconProps>) {
+  return (
+    // biome-ignore lint/a11y/noSvgWithoutTitle: I think icons should be screen reader ignored, explained via aria-label or button content.
```

Comment:
Hmmm actually, probably worth baking in a default aria-label into each icon so, for the case of icon only buttons, we get screen reader for "free".